### PR TITLE
fix(llm-monitoring): Fix alerts, remove hacky ai_total_tokens_used function

### PR DESCRIPTION
**Pull request by colin-sentry**
Make alerts go to the spansMetrics dataset, and remove the old hack where `sum` didn't work on counters. Also add an alias to avoid hardcoding MRIs.

### CODE REVIEW

File: static/app/views/alerts/rules/metric/utils/getMetricDatasetQueryExtras.tsx [hunk 1]
```diff
@@ -26,10 +26,14 @@ export function getMetricDatasetQueryExtras({
   const disableMetricDataset =
     decodeScalar(location?.query?.disableMetricDataset) === 'true';
 
-  const queryExtras: Record<string, string> =
-    hasMetricDataset && !disableMetricDataset
-      ? {dataset: getMEPAlertsDataset(dataset, newAlertOrQuery)}
-      : {};
+  const queryExtras: Record<string, string> = {};
```
Comment (wmak) [inline static/app/views/alerts/rules/metric/utils/getMetricDatasetQueryExtras.tsx:29]: should split FE and BE prs in two

File: src/sentry/search/events/datasets/spans_metrics.py [hunk 2]
```diff
@@ -31,7 +30,6 @@ def search_filter_converter(
         self,
     ) -> Mapping[str, Callable[[SearchFilter], WhereType | None]]:
         return {
-            "message": self._message_filter_converter,
```
Comment (wmak) [inline src/sentry/search/events/datasets/spans_metrics.py:34]: was this intentionally removed?

Comment (colin-sentry) [reply src/sentry/search/events/datasets/spans_metrics.py:34]: i'm not sure what's going on - some git bug

**Review by gggritso — APPROVED** [review]
LGTM except isn't it supposed to go to indexed spans from the PR description?

**Pull request merged**